packages/jsActions/native-mobile-resources/src/camera/TakePictureAdvanced.ts

```diff
@@ -5,14 +5,76 @@
 // Other code you write will be lost the next time you deploy the project.
 
 import { Big } from "big.js";
-import { Alert, Linking, NativeModules } from "react-native";
-import ImagePicker, { ImagePickerOptions, ImagePickerResponse } from "react-native-image-picker";
+import { Alert, Linking, NativeModules, Platform } from "react-native";
+import {
+    CameraOptions,
+    ErrorCode,
+    ImageLibraryOptions,
+    ImagePickerResponse,
+    launchCamera,
+    launchImageLibrary
+} from "react-native-image-picker";
 import { getLocales } from "react-native-localize";
 
 type PictureSource = "camera" | "imageLibrary" | "either";
 
 type PictureQuality = "original" | "low" | "medium" | "high" | "custom";
 
+type ImagePickerV2Response = {
```

Comment:
So much duplication. These should probably go to something like `types.ts`..

Reply:
Updated